Sources/UIKitExtensions/UITableView+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  UITableViewFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UITableView {
+
+    func setEditing(_ editing: Bool, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UITableView: Base ) in
+            UITableView.setEditing(editing, animated: animated)
+        }
+    }
+
+    func dequeueReusableCell(withIdentifier identifier: String) -> ConfigurationSet<Base> {
+        return set { ( UITableView: Base ) in
+            UITableView.dequeueReusableCell(withIdentifier: identifier)
+        }
+    }
+    
+    @available(iOS 6.0, *)
+    func dequeueReusableCell(withIdentifier identifier: String, for indexPath: IndexPath) -> ConfigurationSet<Base> {
+        return set { ( UITableView: Base ) in
+            UITableView.dequeueReusableCell(withIdentifier: identifier, for: indexPath)
+        }
+    }
+    
+    @available(iOS 6.0, *)
+    func dequeueReusableHeaderFooterView(withIdentifier identifier: String) -> ConfigurationSet<Base> {
+        return set { ( UITableView: Base ) in
+            UITableView.dequeueReusableHeaderFooterView(withIdentifier: identifier)
```

Comment:
This function does not set any properties. it just returns a view. We do not need this function in this context.